documentation/docs/30-advanced/20-hooks.md

```diff
@@ -320,7 +320,7 @@ export async function reroute({ url, fetch }) {
 
 ### transport
 
-This is a collection of _transporters_, which allow you to pass custom types — returned from `load` and form actions — across the server/client boundary. Each transporter contains an `encode` function, which encodes values on the server (or returns `false` for anything that isn't an instance of the type) and a corresponding `decode` function:
+This is a collection of _transporters_, which allow you to pass custom types — returned from `load` and form actions — across the server/client boundary. Each transporter contains an `encode` function, which encodes values on the server (or returns a falsey value for anything that isn't an instance of the type) and a corresponding `decode` function:
```

Comment:
I've always seen it 'falsy', but MDN is saying that 'falsey' is sometimes used https://developer.mozilla.org/en-US/docs/Glossary/Falsy